Assets/MRTK/Core/Utilities/StandardShader/ClippingPrimitive.cs

```diff
@@ -304,13 +363,12 @@ protected virtual void Initialize()
 
         protected virtual void UpdateRenderers()
         {
-            if (renderers == null) { return; }
+            if (renderers.Count == 0) { return; }
```

Comment:
Are we guaranteed that `renderers` will never be null?

Reply:
I think so, but added back the check just in case.